examples/transmitter/swagger_server/swagger/swagger.yaml

```diff
@@ -840,6 +860,51 @@ components:
             Stream Management API calls that require authorization.
       example:
         token: 49e5e7785e4e4f688aa49e2585970370
+    TriggerEventParameters:
+      title: Trigger Event Parameters
+      required:
+      - event_type
+      - subject
+      type: object
+      properties:
+        event_type:
+          type: string
+          description: "Supports all  (not yet supported)[RISC](https://openid.net/specs/openid-risc-event-types-1_0-ID1.html)\
+            \ and (supported)[CAEP](https://openid.net/specs/openid-caep-specification-1_0-ID1.html)\
+            \ event types."
+          example: credential-compromise
+          enum:
+          - session-revoked
+          - token-claims-change
+          - credential-change
+          - assurance-level-change
+          - device-compliance-change
+          - account-purged
+          - account-disabled
+          - account-enabled
+          - identifier-changed
+          - identifier-recycled
```

Comment:
You're missing `credential-compromise` here

Reply:
@FragLegs 
I don't see that event type here https://openid.net/specs/openid-risc-event-types-1_0-ID1.html
Should i refer to some other doc?

Reply:
Ah! i found the latest github page.
https://github.com/openid/sse/blob/main/openid-risc-profile-specification-1_0.html
will Update accordingly.